src/function.rs

```diff
@@ -59,6 +60,17 @@ impl Function {
   }
 }
 
+fn absolute_path(context: &FunctionContext, path: &str) -> Result<String, String> {
+  let abs_path_unchecked = context.search.working_directory.join(path).lexiclean();
+  match abs_path_unchecked.to_str() {
+    Some(wd) => Ok(Utf8Path::new(wd).to_string()),
```

Comment:
We can just call `to_owned` here, `Utf8Path::new` is a noop:

```suggestion
    Some(absolute_path) => Ok(absolute_path.to_owned()),
```